pkg/instance/ttlDestroyConsumer_test.go

```diff
@@ -60,6 +60,10 @@ type instanceService struct {
 	mock.Mock
 }
 
+func (is *instanceService) Restart(token string, id uint) error {
```

Comment:
We should isolate the ttlDestroyConsumer from such changes, since its actually only interested in the ability to destroy an instance.

Don't remember if you already have another PR somewhere to adapt the interface 😄

Reply:
Nope, that was for the api gateway